src/lib/components/ViewBox.svelte

```diff
@@ -178,6 +178,11 @@
     viewBox.height *= zoomFactor;
   }
 
+  export function zooming(zoomFactor: number) {
```

Comment:
rename function to zoom

Reply:
hvis den bare hedder zoom skal handleWheel vel bruge zoom funktionen idk